python-packages/order_utils/test/test_generate_order_hash_hex.py

```diff
@@ -0,0 +1,12 @@
+"""Test zero_ex.order_utils.get_order_hash_hex()."""
+
+from zero_ex.order_utils import generate_order_hash_hex, make_empty_order
+
+
+def test_get_order_hash_hex__empty_order():
+    """Test the hashing of an uninitialized order."""
+    expected_hash_hex = (
+        "faa49b35faeb9197e9c3ba7a52075e6dad19739549f153b77dfcf59408a4b422"
```

Comment:
Here is that usage of the expected value from the new test in `packages/utils/test/sign_typed_data_utils_test.ts`.